cargo-workspaces/src/utils/publish_helpers.rs

```diff
@@ -0,0 +1,106 @@
+//! Helper functions useful when publishing (or preparing for publishing) crates.
+
+use std::convert::TryFrom;
+
+use crate::utils::{cargo_config_get, Error, Result, INTERNAL_ERR};
+
+use camino::Utf8PathBuf;
+use cargo_metadata::{Metadata, Package};
+use indexmap::IndexSet as Set;
+use tame_index::{
+    external::{
+        http::{HeaderMap, HeaderValue},
+        reqwest::{blocking::Client, header::AUTHORIZATION, Certificate},
+    },
+    index::{ComboIndex, ComboIndexCache, RemoteGitIndex, RemoteSparseIndex},
+    utils::flock::LockOptions,
+    IndexLocation, IndexUrl, KrateName,
+};
+
+pub(crate) fn filter_private(
```

Comment:
```suggestion
pub fn filter_private(
```

We don't need to do `pub(crate)` in this file. We can simply do `pub`